handler.js

```diff
@@ -0,0 +1,76 @@
+'use strict';
+const AWS = require('aws-sdk');
+const ccxt = require('ccxt');
+const axios = require('axios');
+
+module.exports.transaction = async (event) => {
+  var jobId = event.Records[0].Sns.Message;
+
+  var jobURL = `http://jobs.service.thinblock.io/jobs/${jobId}?look_up=_id`;
+  const { data: { actions: [ action ] } } = await axios.get(jobURL);
+
+  const { params } = action;
+
+  if (params.user_id === undefined || params.exchange === undefined ||
+    params.symbol === undefined || params.side === undefined ||
+    params.amount === undefined || params.price === undefined) {
+    return {
+      statusCode: 400,
```

Comment:
mmm does it work with returning `json`?

Reply:
I'll be making this change. This will publish the SNS we discussed yesterday.